packages/go_router/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.0.5
```

Comment:
Actually, since this is a test-only change, we can just make this `NEXT`.

Reply:
Ah, very true. I had it as NEXT for a while then convinced myself that I needed to publish this.